docs/source/configuration/slots.md

```diff
@@ -0,0 +1,260 @@
+---
+myst:
+  html_meta:
+    "description": "Slots are insertion points in the Volto rendering tree structure."
+    "property=og:description": "Slots are insertion points in the Volto rendering tree structure."
+    "property=og:title": "Slots"
+    "keywords": "Volto, Plone, frontend, React, configuration, slots, viewlets"
+---
+
+# Slots
+
+Slots are insertion points in the Volto rendering tree structure.
+This concept is inherited from the Plone Classic UI {doc}`plone:classic-ui/viewlets`.
```

Comment:
````suggestion
```{note}
This concept is inspired by the Plone Classic UI {doc}`plone:classic-ui/viewlets`.
```
````